dashboard/app/controllers/pd/professional_learning_landing_controller.rb

```diff
@@ -13,6 +13,8 @@ def index
     surveys_pending_enrollment = Pd::Enrollment.filter_for_survey_completion(enrollments, false)
     last_pending_enrollment = surveys_pending_enrollment && surveys_pending_enrollment.max_by(&:survey_sent_at)
 
+    upcoming_workshops = Pd::Enrollment.where(email: current_user.email, )
```

Comment:
It looks like something is incomplete here. In addition to the lint error that this `upcoming_workshops` is not used, there is nothing after the comma.